website/pages/docs/security/security-models/core.mdx

```diff
@@ -0,0 +1,380 @@
+---
+layout: docs
+page_title: Consul Core Security Model
+sidebar_title: Core
+description: >-
+  Security model including requirements, reccomendations, and threats for the core Consul product.
+---
+
+## Overview 
+
+Consul enables automation of network configurations, service discovery, and secure network connectivity across any 
+cloud or runtime.
+
+Consul utilizes a lightweight gossip and RPC system which provides various essential features. Both of these systems 
+provide security mechanisms which should be utilized to enable confidentiality, integrity and authentication.
+
+Using defense in depth is crucial for Consul security, and deployment requirements may differ drastically depending on 
+your use case. Some security features for multi-tenant deployments are offered exclusively in the enterprise version. 
+This documentation may need to be adapted to your environment, but the general mechanisms for a secure Consul deployment
+revolve around:
+
+- **mTLS** - Mutual authentication of both the TLS server and client x509 certificates prevents internal abuse from 
+  unauthorized access to network components within the cluster.
+
+- **ACLs** - Enable role-based access controls for authenticated connections by granting capabilities for an individual 
+  human, or machine operator identity via an ACL token to authorize actions within a cluster. Optionally, custom 
+  [authentication methods](/docs/security/acl/auth-methods) can be used to enable trusted external parties to authorize
+  ACL token creation.
+
+- **Namespaces (Enterprise Only)** - Read and write operations can be scoped to a logical name to restrict access to 
```

Comment:
Elsewhere in the docs we use this widget instead:
```
<EnterpriseAlert inline />
```
you could try that out locally and see if that still suits this doc.

Reply:
Nice, I think it looks good! Thank you!